djangoappengine/storage.py

```diff
@@ -114,26 +180,51 @@ def get_valid_name(self, name):
     def get_available_name(self, name):
         return name.replace('\\', '/')
```

Comment:
I'm not too familiar with what `get_available_name` does. It may not provide any real guarantees when used with cloud storage.